colorenv/README.md

```diff
@@ -0,0 +1,13 @@
+# Coloring Environment
+This environment simulates an N x N grid that starts empty. At every timestep the environment randomly selects an empty position and the agent returns a color it wants to put in that position. The cycle repeats until there are no empty spaces at which point the board is scored. A reward of one is returned if no two of the same colors are adjacent and zero otherwise.
+
+### Running the random agent:
+1. Run `source .venv/bin/activate` to enter the virtual environement
+2. Run `python random_coloring_agent.py`
```

Comment:
I think we should assume all scripts will be run from the main directory (rather than `cd`ing into the different environments). So maybe pull this how to run the agents for the different environments into the top-level readme? Leaving the deeper description of the environments in their individual readmes makes sense though (and linking to them from the main readme).

This will also help remove some of the `sys.path.append("..")` stuff you have to do in `color_tune.py` for example.

Thoughts?